analytics/athena/terraform/plan_with_confirmation.sh

```
@@ -118,11 +118,11 @@ done
 echo ""
 echo "✅ Variables collected:"
 echo "   Project: $project"
-echo "   Environment: $env"
+echo "   Environment: $environment"
 echo "   S3 Logs Prefix: $logs_s3_prefix"
 echo ""
 
 # Run terraform plan with collected variables
-echo "Executing: terraform plan -var=\"project=$project\" -var=\"env=$env\" -var=\"logs_s3_prefix=$logs_s3_prefix\" $@"
+echo "Executing: terraform plan -var=\"project=$project\" -var=\"environment=$environment\" -var=\"logs_s3_prefix=$logs_s3_prefix\" $@"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)